packages/ui/src/paywall-tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import * as Tooltip from "@radix-ui/react-tooltip";
+import type { ReactNode, FC } from "react";
+import { Button } from "./button";
+
+export const PaywallTooltip: FC<{ children: ReactNode }> = ({ children }) => {
```

Comment:
Extracted into ui as we'll use this multiple times.

cc: @juliusmarminge @bgavas